packages/cli/src/commands/logAndroid/logAndroid.js

```diff
@@ -5,26 +5,31 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {spawnSync} from 'child_process';
+import {
+  logkitty,
+  makeTagsFilter,
+  formatEntry,
+  formatError,
+  Priority,
+} from 'logkitty';
 import {logger} from '@react-native-community/cli-tools';
 
-/**
- * Starts adb logcat
- */
 async function logAndroid() {
-  const adbPath = process.env.ANDROID_HOME
-    ? `${process.env.ANDROID_HOME}/platform-tools/adb`
-    : 'adb';
-
-  const adbArgs = ['logcat', '*:S', 'ReactNative:V', 'ReactNativeJS:V'];
+  logger.info('Starting logkitty');
 
-  logger.info(`Starting the logger (${adbPath} ${adbArgs.join(' ')})...`);
+  const emitter = logkitty({
+    platform: 'android',
+    minPriority: Priority.VERBOSE,
+    filter: makeTagsFilter('ReactNative', 'ReactNativeJS'),
+  });
 
-  const log = spawnSync(adbPath, adbArgs, {stdio: 'inherit'});
+  emitter.on('entry', (entry: Entry) => {
```

Comment:
Entry type is not imported but I guess there's no Flow types for this lib

Reply:
also does `entry` event cover all levels of logs?

Reply:
As far as I know, `entry covers it all, except `error`. There are no Flow types for this lib, but for some reason, `Flow` is assuming `any` on undefined types. I'll change it.

Reply:
https://github.com/zamotany/logkitty/blob/master/src/types.ts#L1-L7